ts/screens/preferences/PreferencesScreen.tsx

```diff
@@ -33,16 +44,33 @@ type OwnProps = Readonly<{
   navigation: NavigationScreenProp<NavigationState>;
 }>;
 
-type Props = ReturnType<typeof mapStateToProps> & ReduxProps & OwnProps;
+type Props = OwnProps &
+  LightModalContextInterface &
+  ReturnType<typeof mapStateToProps> &
+  ReduxProps;
 
 type State = {
   isFingerprintAvailable: boolean | undefined;
```

Comment:
why we need the `undefined` state? can't we just have both `isFingerprintAvailable` and `hasCalendarPermission` as booleans, with default to `false` ?

Reply:
Yes we can do so as well

Reply:
thanks, developers are happier with less undefined stuff :D